cmd/prompt.go

```diff
@@ -63,7 +63,7 @@ var promptModeCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		err := generateYangSchema(promptDirs, promptFiles, promptExcluded)
 		if err != nil {
-			if !viper.GetBool("log") {
```

Comment:
this is the intended behavior, print the error to stderr if the logger is not initialized.
I think what is missing here is a `logger.Printf("failed to load paths from yang: %v:", err)` before the if statement.